### PR TITLE
chore: update GitHub branch references to use HEAD

### PR DESCRIPTION
**Pull request by thinkingserious**
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ The Web API v3 client is `sendgrid-ruby.rb`
 
 ## Testing
 
-All PRs require passing tests before the PR will be reviewed. All test files are in the [`tests`](https://github.com/sendgrid/sendgrid-ruby/tree/master/test) directory. For the purposes of contributing to this repo, please update the [`test_sendgrid-ruby.rb`](https://github.com/sendgrid/sendgrid-ruby/blob/master/test/test_sendgrid-ruby.rb) file with unit tests as you modify the code.
+All PRs require passing tests before the PR will be reviewed. All test files are in the [`tests`](test) directory. For the purposes of contributing to this repo, please update the [`test_sendgrid-ruby.rb`](test/test_sendgrid-ruby.rb) file with unit tests as you modify the code.
```
Comment (childish-sambino) [inline CONTRIBUTING.md:129]: Link not working.

File: README.md [hunk 2]
```diff
@@ -171,29 +171,29 @@ puts response.headers
...
-- [v3 Web API Mail Send Helper](https://github.com/sendgrid/sendgrid-ruby/tree/master/lib/sendgrid/helpers/mail) - build a request object payload for a v3 /mail/send API call.
-- [Settings Helper](https://github.com/sendgrid/sendgrid-ruby/tree/master/lib/sendgrid/helpers/settings)
+- [v3 Web API Mail Send Helper](lib/sendgrid/helpers/mail) - build a request object payload for a v3 /mail/send API call.
+- [Settings Helper](lib/sendgrid/helpers/settings)
 
 <a name="use_cases"></a>
 # Use Cases
 
-[Examples of common API use cases](https://github.com/sendgrid/sendgrid-ruby/blob/master/USE_CASES.md), such as how to send an email with a transactional template.
+[Examples of common API use cases](USE_CASES.md), such as how to send an email with a transactional template.
```
Comment (childish-sambino) [inline README.md:189]: Link not working.

File: lib/sendgrid/helpers/inbound/README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@
 # Installation
 
 In addition to the installation instructions in
-[the main readme](https://github.com/sendgrid/sendgrid-ruby/tree/master/#installation),
+[the main readme](#installation),
```
Comment (childish-sambino) [inline lib/sendgrid/helpers/inbound/README.md:20]: No links are working.

Comment (thinkingserious) [reply lib/sendgrid/helpers/inbound/README.md:20]: [These links](https://github.com/sendgrid/sendgrid-ruby/blob/09752f9d90c68ec89d7fb35ed7b0f4757026d39f/lib/sendgrid/helpers/inbound/README.md) are all working for me.

Comment (childish-sambino) [reply lib/sendgrid/helpers/inbound/README.md:20]: Then the link makes no sense. It's referencing the anchor from 3 lines up?

File: lib/sendgrid/helpers/inbound/README.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ bundle install
 ruby ./lib/sendgrid/helpers/inbound/send.rb ./lib/sendgrid/helpers/inbound/sample_data/default_data.txt
 ```
 
-More sample data can be found [here](https://github.com/sendgrid/sendgrid-ruby/tree/master/lib/sendgrid/helpers/inbound/sample_data).
+More sample data can be found [here](lib/sendgrid/helpers/inbound/sample_data).
```
Comment (childish-sambino) [inline lib/sendgrid/helpers/inbound/README.md:50]: Links still not working.

File: lib/sendgrid/helpers/inbound/README.md [hunk 5]
```diff
@@ -82,11 +82,11 @@ Next, send an email to [anything]@inbound.yourdomain.com, then look at the termi
 
 ## app.rb
 
-This module runs a [Sinatra](http://www.sinatrarb.com/) server, that by default (you can change those settings [here](https://github.com/sendgrid/sendgrid-ruby/blob/master/sendgrid/helpers/inbound/config.yml)), listens for POSTs on http://localhost:9292. When the server receives the POST, it parses and prints the key/value data.
+This module runs a [Sinatra](http://www.sinatrarb.com/) server, that by default (you can change those settings [here](sendgrid/helpers/inbound/config.yml)), listens for POSTs on http://localhost:9292. When the server receives the POST, it parses and prints the key/value data.
 
 ## config.yml
 
-This module loads application environment variables (located in [config.yml](https://github.com/sendgrid/sendgrid-ruby/blob/master/sendgrid/helpers/inbound/config.yml)).
+This module loads application environment variables (located in [config.yml](sendgrid/helpers/inbound/config.yml)).
```
Comment (childish-sambino) [inline lib/sendgrid/helpers/inbound/README.md:89]: Multiple links not working.

File: lib/sendgrid/helpers/inbound/README.md [hunk 6]
```diff
@@ -95,4 +95,4 @@ This module is used to send sample test data. It is useful for testing and devel
 <a name="contributing"></a>
 # Contributing
 
-If you would like to contribute to this project, please see our [contributing guide](https://github.com/sendgrid/sendgrid-ruby/blob/master/CONTRIBUTING.md). Thanks!
+If you would like to contribute to this project, please see our [contributing guide](CONTRIBUTING.md). Thanks!
```
Comment (childish-sambino) [inline lib/sendgrid/helpers/inbound/README.md:98]: Link not working.

**Pull request merged**